dojo/db_migrations/0192_add_finding_verified_date.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.1.13 on 2023-11-14 17:22
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
We need to ensure when we pull from the upstream `dev` branch, the next migration depends on `0192_add_finding_verified_date`, so that the chain is maintained.